src/components/AddComments.jsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { useState, useEffect } from "react";
+import { postArticleVote } from "../utils/api";
+
+export default function AddComments({ article_id }) {
+  const [isPosted, setIsPost] = useState(false);
+  const [newComment, setNewComment] = useState({
+    username: "",
+    body: "",
+  });
+
+  useEffect(() => {
+    setIsPost(false);
+    if (newComment.username !== "") {
+      postArticleVote(article_id, newComment)
+        .then((res) => alert("Posted successfully. Please refresh the page."))
+        .catch((err) => {
+          alert("Please enter a correct username or valid comments. Please refresh the page.");
+        });
+      setIsPost(true);
```

Comment:
If we want to set isPosted to true after a post has happened successfully, this line should go in the .then block that follows a successful post